common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -93,6 +102,12 @@ export interface IDocumentMessage {
 
     // Traces related to the packet.
     traces?: ITrace[];
+
```

Comment:
I believe you want to make same change for ISequencedDocumentMessage.

Reply:
You're correct, this was opened quickly to start discussion. I'll update the PR now to include the property  ISequencedDocumentMessage along with the changes where Gary has pointed us to